fluentforms/core/src/main/java/com/_4point/aem/fluentforms/impl/assembler/AssemblerResultImpl.java

```diff
@@ -0,0 +1,17 @@
+package com._4point.aem.fluentforms.impl.assembler;
+
+import java.util.Map;
+
+import com._4point.aem.fluentforms.api.Document;
+import com._4point.aem.fluentforms.api.assembler.AssemblerResult;
+
+public class AssemblerResultImpl implements AssemblerResult {
+    
+	private Map<String, Document> getDocuments;  
+	
+	@Override
+	public Map<String, Document> getDocuments() {
+			return getDocuments;
+	}
+
+}
```

Comment:
It doesn't appear that this class is being used.  Instead AdobeAssemblerResultImpl seems to be being used, however I would argue that this is the better approach (i.e. use this class and ditch AdobeAssemblerResultImpl).  The goal of AdobeAssemblerServiceAdapter should be to encapsulate the dependencies on the Adobe libraries/API.  The current AdobeAssemblerResultImple breaks that encapsulation by containing the Adobe AssemblerResult object.  I would prefer to see the Adobe object converted to our internal implementation within the AdobeAssemblerServiceAdapter.  That way, the Adobe objects don't persist outside the Adobe...Adapter.